### ISSUE TITLE
Secondary axis label

### ISSUE THREAD

**Issue by mindv0rtex** [epezent/implot#162]
Is it possible to add the secondary or tertiary axis label to the plot? Could I fake it somehow without too much hacking of the internals?..

**Comment by epezent** [comment]
It's not currently possible. We could support it by modifying `BeginPlot` as shown below, but it will take some tinkering to get it working smoothly. Please submit a PR if you decide to take a swing at it.

```cpp
IMPLOT_API bool BeginPlot(const char* title_id,
                          const char* x_label      = NULL,
                          const char* y_label      = NULL,
                          const ImVec2& size       = ImVec2(-1,0),
                          ImPlotFlags flags        = ImPlotFlags_None,
                          ImPlotAxisFlags x_flags  = ImPlotAxisFlags_None,
                          ImPlotAxisFlags y_flags  = ImPlotAxisFlags_None,
                          ImPlotAxisFlags y2_flags = ImPlotAxisFlags_NoGridLines,
                          ImPlotAxisFlags y3_flags = ImPlotAxisFlags_NoGridLines,
                          const char* y2_label     = NULL,
                          const char* y3_label     = NULL); 
```

**Comment by mindv0rtex** [comment]
Getting there...
![Screen Shot 2021-01-05 at 15 05 56](https://user-images.githubusercontent.com/10409817/103693590-89f9b000-4f67-11eb-813f-8a203f0991ce.png)

**Comment by mindv0rtex** [comment]
Resolved in #163.

**Comment by epezent** [comment]
#163 has been merged. Thanks Yan!
